packages/vite/src/node/config.ts

```diff
@@ -144,6 +144,11 @@ export interface UserConfig {
    * Preview specific options, e.g. host, port, https...
    */
   preview?: PreviewOptions
+  /**
+   * Force dep pre-optimization regardless of whether deps have changed.
+   * @experimental
+   */
+  force?: boolean
```

Comment:
Force also clears the http cert though

Reply:
Interesting, it looks like then we removed that "feature" when we started deleting `cacheDir/deps` instead of `cacheDir`. We were talking with Anthony that "force" may not be a good name for this. I can move it to `optimizeDeps.cache: boolean | 'build' | 'dev'` (default to true) maybe? Or `optimizeDeps.devCache` and `optimizeDeps.buildCache`? I kind of like the separated option here.
But I really don't know... maybe we need a new shared `cache: { dev { .... }, build: { .... } }`. There could be different caches in the future (like moduleGraph). I think it isn't bad to have each feat have its own cache field though. 
What about merging and then discussing this in another PR? The force option is experimental, we need to discuss it in a meeting before release anyways.

Reply:
That's fine by me too, we can merge this first 👍  I'm also not clear how we should organize these new options as features start to overlap, and is definitely something we should discussed first.